### ISSUE TITLE
Collections: Explore Collections and Collection banners issues

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-lux#764]
This ticket relates to the Blacklight upgrade testing from Feb 2024.

When attempting to view Collections in digital-test I have encountered a few issues:

1. The Explore Collections link which sends a search query is not returning Collections, but instead it returns a list of Works.
2. When viewing a Collection landing page, the banner image is not loading.

NOTE: if you access a collection from the homepage > Featured Collections boxes, some of them may be hard-coded to the production URL, so make sure you stay in the test or arch environment.

**Explore Collections link**
Go to the homepage, click on Explore Collections
See the [search results list generated ](https://digital-test.library.emory.edu/?f%5Bhas_model_ssim%5D%5B%5D=Collection&f%5Bvisibility_ssi%5D%5B%5D=open)(Works are shown, but this should only retrieve Collections)
Compare to production behavior: [Explore Collections search results list](https://digital.library.emory.edu/?f%5Bhas_model_ssim%5D%5B%5D=Collection&f%5Bvisibility_ssi%5D%5B%5D=open)

**Collection Landing Page**
Navigate to a Collection which has a banner image assigned, e.g. [Health Sciences Artifacts](https://digital-test.library.emory.edu/catalog/5053ffbg7n-cor)
See missing/broken banner image
Compare to production instance: https://digital.library.emory.edu/catalog/5053ffbg7n-cor

**Comment by bwatson78** [comment]
Fix for the Explore Collections issue is upcoming.

@eporter23 Before I push that up, I see that some of the collection banners loading (https://curate-test.library.emory.edu//branding/706g1jwsv6-cor/banner/hero-yellowbacks.jpg), and the one you reference doesn't actually pull an image from the server (https://curate-test.library.emory.edu//branding/5053ffbg7n-cor/banner/HS-S023_B067_P004.jpg). Do you want me to check the server and see where the image is actually located at?

**Comment by bwatson78** [comment]
Commit `45fddd3efa045fbc24874aa69905d1ddeebdf527` addresses the Explore Collections issue.

**Comment by eporter23** [comment]
@bwatson78 I'll take a look, I had only been looking at Lux so I'll check in Curate first to see if banners are wonky there. That has been an issue in the past IIRC.

**Comment by eporter23** [comment]
@bwatson78 I checked curate-test against digital-test, and I can confirm that only 1 collection (Yellowbacks) has a functional banner in curate and in lux. I think these got wiped out at some point while we were doing Hyrax upgrades, and we managed to prevent that from happening in prod. So, I think the issue is probably just the test environment. The Explore Collections link is working as expected now!
